src/applications/vaos/new-appointment/components/ClinicChoicePage/index.jsx

```diff
@@ -61,7 +61,8 @@ export default function ClinicChoicePage() {
     data.clinicId === 'NONE' && !eligibility?.request;
   const usingPastClinics =
     typeOfCare.id !== TYPE_OF_CARE_IDS.PRIMARY_CARE &&
-    (typeOfCare.id !== TYPE_OF_CARE_IDS.MENTAL_HEALTH || usePastVisitMHFilter);
+    (typeOfCare.id !== TYPE_OF_CARE_IDS.MENTAL_HEALTH_SERVICES_ID ||
+      usePastVisitMHFilter);
```

Comment:
Can the name prefixed with 'select'? This is the pattern used for all selectors. The 'use' prefix is typically used for React Hooks.

Reply:
Yes. You are right.